File: backend/src/main/java/com/example/demo/entities/Booking.java
package com.example.demo.entities;

import java.sql.Date;
import java.time.LocalDateTime;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Entity
@Table(name = "bookings")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class Booking {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "booking_id")
    private Long bookingId;

    @Column(name = "req_id")
    private Long reqId;

    @Column(name = "actual_pickup_date")
    private LocalDateTime actualPickupDate;

    @Column(name = "actual_return_date")
    private LocalDateTime actualReturnDate;

    @Column(name = "amount")
    private float amount;

    @Column(name = "payment_mode")
    private String paymentMode;

    @Column(name = "payment_date")
    private Date paymentDate;

    @Column(name = "transaction_id")
    private String transactionId;

    @ManyToOne
    @JoinColumn(name = "req_id", referencedColumnName = "req_id", insertable = false, updatable = false)
    private BookingRequest bookingRequest;
}

File: dot.net/CRMS2/CRMS2/Models/Booking.cs
using System;
using System.Collections.Generic;

namespace CRMS2.Models
{
    public partial class Booking
    {
        public int BookingId { get; set; }
        public int? ReqId { get; set; }
        public DateTime? ActualPickupDate { get; set; }
        public DateTime? ActualReturnDate { get; set; }
        public decimal? Amount { get; set; }
        public string? PaymentMode { get; set; }
        public DateTime? PaymentDate { get; set; }
        public string? TransactionId { get; set; }

        public virtual BookingRequest? Req { get; set; }
    }
}
